### DISCUSSION TITLE
The tag 'Interaction.Triggers' does not exist in XML namespace 'http://schemas.microsoft.com/expression/2010/interactivity'

### DISCUSSION THREAD

**Discussion by jojopeter** [PrismLibrary/Prism#2350] [WPF]
Hi,
I am getting below exception when I ported the freamework to .Net 5.0. I am using Prism.Wpf version 8.0.0.1909
.Net 5.0 version is supported? Am I missing something else? Any help greatly appreciated.

Thanks, Peter

**Comment by brianlagunas** [comment]
We do not have a Nuget release of our .NET 5 support yet. You have to get that from the https://sponsorconnect.dev/ for now. You get access to that by sponsoring myself or @dansiegel at the $10 a month level.
